include/applicationHeader.h

```diff
@@ -0,0 +1,30 @@
+#ifndef APPLICATIONHEADER_H_INCLUDED
+#define APPLICATIONHEADER_H_INCLUDED
+
+//public
+/*
+       Wenn Result.error_code = SUCCESS steht im Result.value der Wert den der 
+       Client auf der Konsole sehen soll.
+       Ansonsten wird nur der error_code im Result �bermittelt und das value
+       bleibt LEER.
+*/
+
+//private
+//Entscheidet ob und welcher Befehl ausgef�hrt wird, gibt Result an ClientSession zur�ck. M�gliche Error: COMMAND_NOT_FOUND, MISSING_ARGUMENT, TO_MANY_ARGUMENTS
+struct Result executeCommand(struct Command command);
+
+//Gibt Key und Value an Datenhaltung zur speicherung weiter, gibt Result an ClientSession zur�ck.
+struct Result Put(struct Command comm);
+
+//Fragt den Value zum gegebenen Key an, gibt Result an ClientSession zur�ck. M�gliche Error: KEY_NOT_FOUND
+struct Result get(struct Command command);
+
+//Gibt Key zur L�schung an Datenhaltung weiter, gibt Result an ClientSession zur�ck. M�gliche Error: KEY_NOT_FOUND
+struct Result del(struct Command command);
+
+//Beendet die Verbindung des ScoektClient
+void quit(struct Command command);
```

Comment:
These are internal functions. They must not be declared outside the scope of this file.
Also it would be "put" instead of "Put".